operations/iuf/stages/management_nodes_rollout.md

```diff
@@ -218,3 +220,85 @@ or [Upgrade CSM and additional products with IUF](../workflows/upgrade_csm_and_a
+These values will be needed when upgrading the NCN storage nodes in the following steps.
+
+1. Perform the NCN storage node upgrades.
+
+    1. Set the CFS configuration on all storage nodes.
+
+        1. (`ncn-m#`) Set `CFS_CONFIG_NAME` to be the value for `configuration` found for `Management_Storage` nodes in the previous step.
+
+            ```bash
+            CFS_CONFIG_NAME=<appropriate configuration value>
+            ```
+
+        1. (`ncn-m#`) Get all NCN storage node xnames.
+
+            ```bash
+            XNAMES=$(cray hsm state components list --role Management --subrole Storage --type Node --format json | jq -r '.Components | map(.ID) | join(",")')
+            echo "${XNAMES}"
+            ```
+
+        1. (`ncn-m#`) Set the configuration on all storage nodes.
+
+            ```bash
+            /usr/share/doc/csm/scripts/operations/configuration/apply_csm_configuration.sh \
+            --no-config-change --config-name "${CFS_CONFIG_NAME}" --xnames "${XNAMES}" --no-enable --no-clear-err
+            ```
+
+            The expected output is:
+
+              ```bash
+              All components updated successfully.
+              ```
+
+    1. Set the image in BSS for all storage nodes by following the [Set NCN boot image for `ncn-m001` and NCN storage nodes](../stages/management_nodes_rollout.md#set-ncn-boot-image-for-ncn-m001-and-ncn-storage-nodes)
+    section of this document.
+    Set the `IMS_RESULTANT_IMAGE_ID` variable to the `final_image_id` value for `Management_Storage` found in step 2 above.
+
+    1. (`ncn-m#`) Upgrade one NCN storage node (`ncn-s001`).
+
+        **NOTE** This creates an additional, separate Argo workflow for rebuilding a NCN storage node. The Argo workflow name will include the string `ncn-lifecycle-rebuild`. If monitoring progress with the Argo UI, remember to include these workflows.
+
+        ```bash
+        /usr/share/doc/csm/upgrade/scripts/upgrade/ncn-upgrade-worker-storage-nodes.sh ncn-s001 --upgrade
+        ```
+
+    1. (`ncn-m#`) Verify that the storage node booted and is configured correctly. The CFS configuration can be
+    verified with the command below using the xname of the node that was upgraded instead of the example value `x3000c0s13b0n0`.
+
+        ```bash
+        XNAME=x3000c0s13b0n0
+        cray cfs components describe "${XNAME}"
+        ```
+
+        The desired value for `configurationStatus` is `configured`. If it is `pending`, then wait for the status to change to `configured`.
+
+    1. (`ncn-m#`) Upgrade the remaining storage nodes serially.
+
+        **NOTE** This creates an additional, separate Argo workflow for upgrading NCN storage nodes. The Argo workflow name will include the string `ncn-lifecycle-rebuild`. If monitoring progress with the Argo UI, remember to include these workflows.
+
+        ```bash
+        /usr/share/doc/csm/upgrade/scripts/upgrade/ncn-upgrade-worker-storage-nodes.sh ncn-s002,ncn-s003,ncn-s004 --upgrade
```

Comment:
Not all sites have exactly 4 storage nodes, right?  I think we should either call out that they might need to edit the command so it matches their system or switch to a comma separated list like it done elsewhere (although the logical would have to omit s001)

Reply:
Added additional instruction to specify storage nodes in comma separated list